frontend/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "HangHub",
-  "version": "1.0.1",
-  "description": "HangHub is a productivity tool that allows you to see users who are working on the same GitHub issue as you. It is available as a browser extension for Chrome and Firefox.",
-  "main": "src/index.js",
-  "scripts": {
-    "build": "webpack --mode=production",
-    "lint": "eslint ./src"
-  },
+  "name": "hang-hub-frontend",
+  "version": "0.0.1",
```

Comment:
Why `0.0.1`?

Reply:
We don't publish this file. I don't see any reason to update this file later.